src/cosmosdb-preview/setup.py

```diff
@@ -16,7 +16,7 @@
 
 # TODO: Confirm this is the right version number you want and it matches your
 # HISTORY.rst entry.
-VERSION = '0.26.0'
+VERSION = '0.27.0'
```

Comment:
There're breaking changes in this PR so you may reconsider the version

Reply:
See https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md